kore/src/Kore/Log/InfoAttemptUnification.hs

```diff
@@ -41,6 +41,7 @@ instance SOP.HasDatatypeInfo InfoAttemptUnification
 instance Entry InfoAttemptUnification where
     entrySeverity _ = Info
     shortDoc _ = Just "while attempting unification"
+    helpDoc _ = "log unification attempts"
```

Comment:
Is this just the start of every unification problem? Or is every sub-expression also logged? Is this only used for rules?